### DISCUSSION TITLE
issue with weather maps

### DISCUSSION THREAD

**Discussion by tikondrus** [home-assistant/frontend#10005] [Dashboard] [answered]
I've configured simple weather maps
         - type: weather-forecast
           entity: weather.home_assistant
           name: Met.no
         - type: weather-forecast
           entity: weather.openweathermap
         - type: weather-forecast
           entity: weather.gismeteo
         - type: weather-forecast
           entity: weather.home_assistant_hourly
           name: Met.no

But as a result I see different day naming in each weather card
![2021-09-10_14-34-59](https://user-images.githubusercontent.com/34535914/132847578-29027f09-e93c-41ea-b705-a1a8613276cf.png)

**Comment by spacegaier** [comment, marked as answer]
That this is a backend topic, since the different weather integrations provide the forecast info.

You can check that in the HA dev tools on the "State" tab by selecting the different weather entities and you will see the `datetime` that the forecast belongs to.

**Comment by tikondrus** [comment]
for weather.openweathermap there is ony one entity
weather.home_assistant also has weather.home_assistant_hourly, but I have issue with the first one
weather.gismeteo aslo has  only one entity

**Comment by tikondrus** [comment]
oh, I got it. datetime: has different dates for different entities. It looks like not a UI problem. Thanks a lot!
